aws-xray-recorder-sdk-core/src/main/java/com/amazonaws/xray/entities/Subsegment.java

```diff
@@ -96,10 +96,16 @@ static Subsegment noOp(Segment parent, AWSXRayRecorder recorder) {
      * own. Only used for debugging.
      *
      * @return
-     *  the pretty string representation of the subsegment with enouogh information for it to be streamed
+     *  the pretty string representation of the subsegment with enough information for it to be streamed
      */
     String prettyStreamSerialize();
 
+    /**
+     * Implements the {@link AutoCloseable} interface.
+     *
+     * Warning. This method is intended to private to the xray library and should not be used externally. Instead, use
```

Comment:
Small nitpick - This method is intended _be_ to private

Reply:
Done